core-bundle/src/Command/UserCreateCommand.php

```diff
@@ -0,0 +1,296 @@
+        );
+
+        $io->success(sprintf('User %s%s created.', $username, $isAdmin ? ' with admin permissions' : ''));
+
+        return 0;
+    }
+
+    private function ask(string $label, InputInterface $input, OutputInterface $output, callable $callback = null): string
+    {
+        $question = new Question($label);
+        $question->setMaxAttempts(3);
+        $question->setValidator($callback);
+
+        /** @var QuestionHelper $helper */
+        $helper = $this->getHelper('question');
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function askForPassword(string $label, InputInterface $input, OutputInterface $output, callable $callback): string
+    {
+        $question = new Question($label);
+        $question->setHidden(true);
+        $question->setMaxAttempts(3);
+        $question->setValidator($callback);
+
+        /** @var QuestionHelper $helper */
+        $helper = $this->getHelper('question');
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function askChoice(string $label, array $options, InputInterface $input, OutputInterface $output): string
+    {
+        $question = new ChoiceQuestion($label, $options);
+        $question->setAutocompleterValues($options);
+
+        /** @var QuestionHelper $helper */
+        $helper = $this->getHelper('question');
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function askMultipleChoice(string $label, array $options, InputInterface $input, OutputInterface $output): array
+    {
+        $question = new ChoiceQuestion($label, $options);
+        $question->setAutocompleterValues($options);
+        $question->setMultiselect(true);
+
+        /** @var QuestionHelper $helper */
+        $helper = $this->getHelper('question');
+
+        return $helper->ask($input, $output, $question);
+    }
+
+    private function getGroups(): array
+    {
+        $this->framework->initialize();
+
+        /** @var UserGroupModel $userGroupModel */
+        $userGroupModel = $this->framework->getAdapter(UserGroupModel::class);
+        $groups = $userGroupModel->findAll();
+
+        if (null === $groups) {
+            return [];
+        }
+
+        return $groups->fetchEach('name');
+    }
+
+    private function persistUser(string $username, string $name, string $email, string $password, string $language, bool $isAdmin = false, array $groups = null, bool $pwChange = false): void
+    {
+        $time = time();
+        $hash = $this->encoderFactory->getEncoder(BackendUser::class)->encodePassword($password, null);
+
+        $this->connection->insert(
+            'tl_user',
+            [
+                'tstamp' => $time,
+                'name' => $name,
+                'email' => $email,
+                'username' => $username,
+                'password' => $hash,
+                'language' => $language,
+                'backendTheme' => 'flexible',
```

Comment:
@leo shouldn't this default be in the SQL definition of `tl_user`?

Reply:
It probably should, but it currently is not.